src/serializers/type_serializers/datetime_etc.rs

```diff
@@ -25,6 +39,14 @@ pub(crate) fn time_to_string(py_time: &Bound<'_, PyTime>) -> PyResult<String> {
     pytime_as_time(py_time, None).map(|dt| dt.to_string())
 }
 
+pub(crate) fn time_to_seconds(py_time: &Bound<'_, PyTime>) -> PyResult<f32> {
+    pytime_as_time(py_time, None).map(|t| t.total_seconds() as f32)
+}
+
+pub(crate) fn time_to_milliseconds(py_time: &Bound<'_, PyTime>) -> PyResult<f32> {
+    pytime_as_time(py_time, None).map(|t| t.total_ms() as f32)
+}
```

Comment:
I feel like again here given we return floats we should be using similar functions to the `EitherTimedelta` stuff, to add the microseconds on as a fraction.